topics/RDMA.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+description: Valkey Over RDMA support
+---
+
+Valkey supports the RDMA connection type as a module that requires compilation
+as a shared library and dynamic loading on demand.
+
+## Getting Started
+
+RDMA stands for Remote Direct Memory Access. It is a technology that allows computers in
+a network to exchange data directly in the main memory, bypassing the involvement of processors,
+caches, or operating systems on either computer.
+As a result, RDMA offers better performance compared to TCP. Test results indicate that
+Valkey Over RDMA achieves approximately 2 times higher QPS and lower latency.
+
+Please note that Valkey Over RDMA is currently supported only on Linux.
+
+## Running manually
+
+To manually run a Valkey server with RDMA mode:
+
+    ./src/valkey-server --protected-mode no \
+         --loadmodule src/valkey-rdma.so bind=192.168.122.100 port=6379
+
+It's possible to change bind address/port of RDMA by runtime command:
```

Comment:
```suggestion
RDMA's bind address/port can be modified at runtime using the following command:
```

Reply:
Use `Bind address/port of RDMA can be modified at runtime using the following command:` instead to avoid `RDMA's` spellcheck error.